odoo_backup_sh/security/security_groups.xml

```diff
@@ -1,10 +1,16 @@
 <?xml version="1.0"?>
 <odoo>
+    <record model="ir.module.category" id="backup_category">
+        <field name="name">Backup</field>
+    </record>
+
     <record model="res.groups" id="group_manager">
         <field name="name">Backup manager</field>
+        <field name="category_id" ref="backup_category" />
```

Comment:
Manage has to inherit user rights (implied_ids field or something)

Reply:
Added implied_ids below: https://github.com/it-projects-llc/addons-dev/pull/717/commits/543cd712902417d8136a63dccc57a7efa00b0443